src/lib/services/move/module/lcd.ts

```diff
@@ -6,26 +6,26 @@ import type { Addr, IndexedModule, Nullable } from "lib/types";
 import { parseWithError } from "lib/utils";
 
 export const getModuleByAddressLcd = async (
-  baseEndpoint: string,
-  address: Addr,
+  endpoint: string,
+  vmAddress: Addr,
```

Comment:
If the endpoint supports both hex and bech, then keep it as address.

If not, change the type Addr to HexAddr